cricviz/Gemfile

```diff
@@ -0,0 +1,58 @@
+source 'https://rubygems.org'
```

Comment:
You don't need to create your own rails project - edit the existing `session_2/cricviz` folder.

Reply:
I don't seem to have an existing file session_2/cricviz on the clone repo IRIS-RoR-Bootcamp-2020

Reply:
Sync your fork: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

Reply:
thank you